tests/slurm.config.yml

```diff
@@ -1 +1,4 @@
 Compute: "slurm"
+Slurm:
+  DisableReconciler: false
```

Comment:
Sorry to be picky, but these should match the actual default config, i.e. they should be true.